src/overlays/actors/ovl_Obj_Lightswitch/z_obj_lightswitch.c

```diff
@@ -5,17 +5,39 @@
  */
 
 #include "z_obj_lightswitch.h"
+#include <vt.h>
 
 #define FLAGS 0x00000010
 
 #define THIS ((ObjLightswitch*)thisx)
 
+typedef enum { FACE_EYES_CLOSED, FACE_EYES_OPEN, FACE_EYES_OPEN_SMILING } FaceTextureIndex;
```

Comment:
I guess it won't align vertically with so few items unless we have offset comments. I would add them myself since it's useful anyway, but I'll leave it up to you and the others to decide.
```suggestion
typedef enum {
    /* 0x00 */ FACE_EYES_CLOSED, 
    /* 0x01 */ FACE_EYES_OPEN, 
    /* 0x02 */ FACE_EYES_OPEN_SMILING
} FaceTextureIndex;
```

Reply:
Ive personally been putting them in enums. so we can do it at least for the sake of consistency for now and decide later what we wanna do